CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `Set` stdlib
 - `Option.force_msg`
+- Assign patterns to variables
```

Comment:
An example snippet might be nice!?

Reply:
Updated docs and changelog in this commit 6c53144ccb14830846d63ed3bfe6f3df4eda0ad8